ocicrypt-rs/build.rs

```diff
@@ -37,7 +37,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             .include("src/utils")
             .rust_protobuf()
             .customize(ttrpc_codegen::Customize {
-                async_all: true,
+                async_all: false,
```

Comment:
Hi @arronwy , I changed the ttrpc client to sync here. Please take a look if it is good to you